src/wp-includes/class-wp-plugin-dependencies.php

```diff
@@ -359,10 +359,16 @@ public static function get_dependency_data( $slug ) {
 	 */
 	public static function display_admin_notice_for_unmet_dependencies() {
 		if ( in_array( false, self::get_dependency_filepaths(), true ) ) {
+			$plugins_url = is_multisite() ? network_admin_url( 'plugins' ) : admin_url( 'plugins' );
 			wp_admin_notice(
-				__( 'There are additional plugin dependencies that must be installed.' ),
+				sprintf(
+					/* translators: 1: The start of a link to the plugins page, 2: The end of the link. */
+					__( 'Some required plugins are missing. Please %1$sreview your website&#8217;s plugins%2$s for plugins that cannot be activated yet.' ),
```

Comment:
Is the word "website's" really needed here? I think that "Some required plugins are missing. Please review your plugins for plugins that cannot be activated." would be sufficient. I think that 'website' can be assumed, since there is no other possible context, and the time frame of "the present" can also be assumed.

Reply:
Thanks Joe! Done in [fd17150](https://github.com/WordPress/wordpress-develop/pull/6070/commits/fd1715074234b1ff0727faee20c335fa0d09b0af).